06-Customizing AzTS for your org/Extending AzTS/ManagementGroupBasedComplianceInitiative.md

```diff
@@ -67,11 +68,70 @@ By default, this feature is not turned on. If you haven't enabled it in your AzT
 10. Click 'OK' and then click 'Save'.
 ![Resources](../../Images/06_MGCIEnableMGAndSave.png)
 
-### 3.	Add Custom tags to controls (in bulk edit mode) for compliance initiative
+-----
+### Option2: Using Helper Script
```

Comment:
Can we call this "Using Powershell" or something better?

Reply:
Changed to Using Powershell